functions-and-scope.js

```diff
@@ -42,6 +61,26 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6.642857142857143
 
+//De stappen:
+// 1. Om het gemiddelde te berekenen deel je de totale som door het aantal getallen.
+// 2. Ik wil dus een variabele om de totale som in op te slaan, en een variabele om te weten hoeveel getallen we hebben gehad.
+// 3. Wanneer de hele array doorlopen is kan ik het totaal delen door het aantal stappen, en return ik de uitkomst.
+
+function averageGrade(allGrades) {
+    let counter = 0;
+    let total = 0;
+
+    for (let i = 0; i <= (allGrades.length - 1); i++) {
+       counter = counter + 1;
+       total = total + allGrades[i];
```

Comment:
Dit kun je versimpelen:
```suggestion
       total += allGrades[i];
```